<?xml version="1.0"?>
<launch>

  <arg name="save_name" default="performance.txt"/>
  <arg name="save_location" default="/home/autosoftlab/Desktop/RobotTestGeneration"/>

  <arg name="namespace" default="bebop" />
  <arg name="ip" default="192.168.42.1" />
  <!-- <arg name="ip" default="10.202.0.1" />-->
  <arg name="drone_type" default="bebop2" /> <!-- available drone types: bebop1, bebop2 -->
  <arg name="config_file" default="$(find bebop_driver)/config/defaults.yaml" />
  <arg name="camera_info_url" default="package://bebop_driver/data/$(arg drone_type)_camera_calib.yaml" />

  <node pkg="bebop_utils" type="performance_analyzer.py" name="performance_tester_node" output="screen">
    <param name="save_location" type="string" value="$(arg save_location)"/>
    <param name="save_name" type="string" value="$(arg save_name)"/>
  </node>

  <include file="$(find bebop_driver)/launch/bebop_node.launch">
    <arg name="namespace" default="$(arg namespace)" />
    <arg name="ip" default="$(arg ip)" />
    <arg name="drone_type" default="$(arg drone_type)" />
    <arg name="config_file" default="$(arg config_file)" />
    <arg name="camera_info_url" default="$(arg camera_info_url)" />
  </include>

</launch>
